src/connectedk8s/azext_connectedk8s/_params.py

```diff
@@ -34,6 +34,9 @@ def load_arguments(self, _):
         c.argument('infrastructure', options_list=['--infrastructure'], help='The infrastructure on which the Kubernetes cluster represented by this connected cluster will be running on.', arg_type=get_enum_type(Infrastructure_Enum_Values))
         c.argument('disable_auto_upgrade', options_list=['--disable-auto-upgrade'], action='store_true', help='Flag to disable auto upgrade of arc agents.')
         c.argument('cl_oid', options_list=['--custom-locations-oid'], help="OID of 'custom-locations' app")
+        c.argument('container_registry_repository', options_list=['--registry-repository'], help='Custom container registry repository.')
+        c.argument('container_registry_username', options_list=['--registry-username'], help='Custom container registry repository username.')
```

Comment:
Also, it would be better to group these 3 new arguments by specifying 'arg_grp=<private_container r_registry_arguments>' (or any other name that makes sense). As an exapmple, you can take a look at how the proxy params are grouped

Reply:
+1 I like this approach for usability

Reply:
Ack